Script to purge dead grapl-plugin jobs from Nomad

twunderlich-grapl opened the pull request:
<!-- Thank you for your contribution to Grapl! Please take some time
to fill out this short template to help us review and understand your
PR. -->

### Which issue does this PR correspond to?
Fixes https://github.com/grapl-security/issue-tracker/issues/1044
<!-- Please provide a link to the GitHub Issue this pull request is
meant to address. -->

### What changes does this PR make to Grapl? Why?
Script to clean up dead grapl-plugin jobs from Nomad.
We're seeing a number of dead grapl-plugin jobs, all in different namespaces. Even though these jobs are dead, they're not cleaned up automatically. As such, we need a script to clean these up

<!-- Please describe the functional impact of these changes and the
rationale behind them. This will help us understand your approach to
solving the problem. -->

### How were these changes tested?
Ran the script
<!-- Please describe how these changes were tested. There should be
sufficient detail that reviewers can replicate your testing
procedure. If the procedure is a manual one that's OK, your
description will help us work with you to get it automated. -->

Commit:
Script to purge dead grapl-plugin jobs from Nomad

We're seeing a number of dead grapl-plugin jobs, all in different
namespaces. Even though these jobs are dead, they're not cleaned up
automatically. As such, we need a script to clean these up

christophermaier commented on bin/nomad_plugin_purge.sh:
```diff
@@ -0,0 +1,17 @@
+# Make sure there is an open tunnel to Nomad (typically via ./bin/aws/ssm_nomad_server.sh in a separate tab)
+# Then run this script ./bin/nomad_plugin_purge.sh
+
+plugin_namespaces=$(nomad job stop -namespace=* -purge grapl-plugin 2>&1 | tail -n +3 | awk '{ if ($5 == "dead") { print $2 } }')
```
Does the bare `*` cause shell globbing issues? Should it be single-quoted?

Might also be good to add a typical example of the expected output so the `tail` and `awk` commands make a bit more sense; as it is, I have no idea how to evaluate that pipeline.

twunderlich-grapl replied:
Updated. Thanks for the suggestions, very useful especially to our future selves when hashicorp changes the return

Commit:
Quote the * and document output

Quoting wildcard to prevent globbing, and document the output of the
nomad stop command so that the tail and awk commands make sense

christophermaier reviewed (approved):
Looks great; thanks for the changes! 🙇
